content/en/docs/reference/kubectl/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "kubectl CLI"
+title: "CLI"
```

Comment:
I suggest to revert this change? All pages in this section are related to kubectl. The left navigation lists "CLI" instead of "kubectl CLI"?

Reply:
“kubectl CLI” feels tautological; how about just “kubectl”?

Reply:
Agreed. Was going to suggest that ...